tests/drucker_prager_compression/solution-00000.0000.vtu

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?> 
```

Comment:
vtu files make for poor comparison files because even small variations in the output lead to changes in the binary compressed files. if you want to store output files in the testsuite, please select gnuplot format: this is a text format that is easier to compare for small numerical variations.